lib/pagination-item/index.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cx } from 'emotion';
+
+import { pageItem, active, disabled } from 'emotion-bootstrap';
+
+const propTypes = {
+  active: PropTypes.bool,
+  children: PropTypes.node,
+  className: PropTypes.string,
+  disabled: PropTypes.bool,
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+};
+
+const defaultProps = {
+  tag: 'li',
+};
+
+const PaginationItem = (props) => {
+  const {
+    active: activeCss,
+    className,
+    disabled: disabledCss,
+    tag: Tag,
+    ...attributes
+  } = props;
+
+  const classes = cx(
+    className,
+    pageItem,
+    activeCss && active,
```

Comment:
Putting active in classes using cx mushes everything together, try separating active, mabye something like
```
className={`${classes} ${activeCSS ? active : "")`}
```